src/components/Collection/Entries/EntriesCollection.js

```diff
@@ -35,8 +40,17 @@ class EntriesCollection extends React.Component {
     loadEntries(collection, page);
   }
 
+  cursorHandler = action => () => {
+    const { collection, traverseCollectionCursor } = this.props;
+    traverseCollectionCursor(collection, action);
+  };
+
+  getCursorActionFunctions = (cursor, handler) => cursor && cursor.actions
+    ? cursor.actions.reduce((acc, action) => ({ ...acc, [action]: handler(action) }), {})
+    : {};
+
```

Comment:
Cursor interaction functions like the ones added to this file should be moved to the `Cursor.js` library and imported.